acestep/llm_inference.py

```
@@ -119,22 +120,44 @@ def unload(self) -> None:
             self._mlx_model = None
             self._mlx_model_path = None
+
             gc.collect()
+
             if torch.cuda.is_available():
-                torch.cuda.empty_cache()
-                torch.cuda.synchronize()
+                try:
+                    torch.cuda.synchronize()
+                except Exception:
+                    pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)